bulla-contracts/src/mappings/BullaFactoring.ts

```diff
@@ -248,7 +250,7 @@ export function handleInvoiceUnfactoredV2(event: InvoiceUnfactored): void {
 export function handleDeposit(event: Deposit, version: string): void {
   const ev = event.params;
 
-  const DepositMadeEvent = createDepositMadeEvent(event);
```

Comment:
Works for V2 but not for V1. V1 does not have `Deposit` event, rather `DepositMade` event